### ISSUE TITLE
Training log file for flava full

### ISSUE THREAD

**Issue by rishabhm12** [facebookresearch/multimodal#421]
### 🚀 The feature, motivation and pitch

Hi guys,
If possible, can you attach the training log file of the flava full model in the readme.md section? 
Thank you!

**Comment by ebsmothers** [comment]
Hi @rishabhm12 thanks for the issue. I don't think it makes sense to upload the full training log as it is quite large; also it won't really translate anyways if you are working with a different dataset.

Yes, this is the correct checkpoint to use for fine-tuning. If the image losses are not decreasing as expected you can try modifying your trainer params to account for the different dataset. For instance the learning rate differs for fine-tuning on SNLI and VQA V2 datasets (full details can be found in section B.2 of the [paper](https://arxiv.org/pdf/2112.04482.pdf)).

**Comment by rishabhm12** [comment]
Hi @ebsmothers ,
Thanks for the reply. Sure, I'll play around with diff hyper params and check for the losses again. Closing this issue, will get back as and when I have more questions (if any)
